src/python/foglamp/data_purge/config.json

```diff
@@ -0,0 +1 @@
+{"enabled": true, "lastID": 1998276, "age": "2 minutes", "retainUnsent": true, "wait": "2 days"}
```

Comment:
Ultimately some of this will come from the configuration manager (see FOGL-217) and some will come from the streams information (i.e. the ID of the last reading sent on each stream). In the shorter having this in the file is fine, but the lastID change will have to be done for the August 15 demo integration.